Emby.Server.Implementations/LiveTv/TunerHosts/M3uParser.cs

```diff
@@ -158,15 +158,14 @@ private ChannelInfo GetChannelnfo(string extInf, string tunerHostId, string medi
         private string GetChannelNumber(string extInf, Dictionary<string, string> attributes, string mediaUrl)
         {
             var nameParts = extInf.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries);
-            var nameInExtInf = nameParts.Length > 1 ? nameParts[nameParts.Length - 1].Trim() : null;
+            var nameInExtInf = nameParts.Length > 1 ? nameParts[^1].AsSpan().Trim() : ReadOnlySpan<char>.Empty;
```

Comment:
`^` is new to me. Guess I should read the updates more in-depth.

Reply:
C#8 feature IIRC, it's amazing